app/presenters/publishing_api_presenters/ministerial_role.rb

```diff
@@ -0,0 +1,29 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::MinisterialRole < PublishingApiPresenters::Item
```

Comment:
I understand the need to demystify the code path for presenting these, however I don't think this duplication of code is helpful. I think these need to either subclass `Placeholder`, or the common methods need to be extracted into a module and mixed in.